src/coreclr/nativeaot/docs/wasm.md

```diff
@@ -0,0 +1,4 @@
+# WASM applications configuration with Native AOT
+
+Currently NativeAOT-LLVM supports following additional properties
+- `WasmHtmlTemplate` which specified path to the HTML template within which WASM application would be embeded.

```

Comment:
Instead of a new file, I think it would be better to use [the existing section](https://github.com/dotnet/runtimelab/blob/feature/NativeAOT-LLVM/docs/using-nativeaot/compiling.md#webassembly).